README.md

```diff
@@ -1,6 +1,6 @@
 > Description
 
-This project is a database for vet clinic, enables you to store, and retrieve animals
+This project is a database for vet clinic, enables you to store, and retrieve animals, their types, ages, owners..
```

Comment:
- **[OPTIONAL]** Your README file is incomplete kindly all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo for professional documentation. 👀 📃 
- **[OPTIONAL]** Kindly review the images in your README work as expected for professional documentation. 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/212772291-79c41cda-cd1f-45b0-96a5-4cb2536cbbe2.png)